jest.config.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-undef */
+
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+const { pathsToModuleNameMapper } = require('ts-jest/utils')
+// In the following statement, replace `./tsconfig` with the path to your `tsconfig` file
+// which contains the path mapping (ie the `compilerOptions.paths` option):
+
+const { compilerOptions } = require('./tsconfig')
+
+
+
+module.exports = {
+	preset: 'ts-jest',
+	testEnvironment: 'node',
+	globals: {
+		"ts-jest": {
+			"tsconfig": "tsconfig.json",
+			"diagnostics": true
+		}
+	},
+	// A map from regular expressions to module names that allow to stub out resources with a single module
+	// moduleNameMapper: {
+	//   '^@/(.*)$': '<rootDir>/src/$1',
+	// },
+	moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths , { prefix: '<rootDir>/' } ),
```

Comment:
Would you prefer to back to the old way and have all paths start with `src` instead of `@` ?

Reply:
I don't care that much on this.  Now that you have made the switch to `@` you can leave it if you want.  It would have been easier from the beginning though if this had been done.  Now it takes just as much work to go back.

Reply:
Looking closer at this, I think that there is now a lot of inconsistency created by this.  Either all places where the `src` alias is used should be switched to using `@` or vice versa.  It would actually be easier to drop the `@` alias that you added, and go back to using the `src` alias.  I also noticed that you are not using many of the other aliases provided by Quasar where they should be used.  It may be good for you to review https://quasar.dev/quasar-cli/handling-webpack#webpack-aliases to see what those are.

Furthermore, either the `pathsToModuleNameMapper` should not be used, and instead just add the correct aliases directly to the `moduleNameMapper` option, or install `require-json5` and load the correct aliases from `./node_modules/@quasar/app/tsconfig-preset.json` and pass those to `pathsToModuleNameMapper`.

Adding the aliases directly looks like:
```javascript
module.exports = {
	preset: 'ts-jest',
	testEnvironment: 'node',
	globals: {
		'ts-jest': {
			'tsconfig': 'tsconfig.json',
			'diagnostics': true
		}
	},
	moduleNameMapper: {
		'^src/(.*)$': '<rootDir>/src/$1',
		'^app/(.*)$': '<rootDir>/$1',
		'^components/(.*)$': '<rootDir>/src/components/$1',
		'^layouts/(.*)$': '<rootDir>/src/layouts/$1',
		'^pages/(.*)$': '<rootDir>/src/pages/$1',
		'^assets/(.*)$': '<rootDir>/src/assets/$1',
		'^boot/(.*)$': '<rootDir>/src/boot/$1'
	},
	testRegex: '(/tests/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$',
};
```
and using require-json5 looks like:
```javascript
const { pathsToModuleNameMapper } = require('ts-jest/utils');
const requireJSON5 = require('require-json5');
const { compilerOptions } = requireJSON5('./node_modules/@quasar/app/tsconfig-preset.json');

module.exports = {
	preset: 'ts-jest',
	testEnvironment: 'node',
	globals: {
		'ts-jest': {
			'tsconfig': 'tsconfig.json',
			'diagnostics': true
		}
	},
	moduleNameMapper: pathsToModuleNameMapper(compilerOptions.paths, { prefix: '<rootDir>/' }),
	testRegex: '(/tests/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$',
};
```
For either of these the `paths` option should be deleted entirely from our `tsconfig.json` file.

Reply:
I put a pull request in to your branch for this pull request that implements the second option, and eliminates the `@` alias.

Reply:
Note it also fixes the eslint issues in the jest.config.js file and the quasar.config.js file.

Reply:
Thanks.  Just merged in.  Still working on this...